challenges/arrayShift/arrayShift.test.js

```
@@ -0,0 +1,37 @@
+const insertShiftArray = (arr, val) => {
+  if (!Array.isArray(arr)) {
+    return 'Not an array';
```

This is a place I would probably throw an error at. The app is likely going to break anyway if you're expecting to receive an array and were returned a string. So instead I would probably put `Throw 'Not an array'` instead of returning